packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5455,6 +5472,8 @@ export default class Meeting extends StatelessWebexPlugin {
       resourceId
     );
 
+    this.isMoveTo = true;
```

Comment:
is it possible for `MeetingUtil.joinMeetingOptions()` to succeed but for us to never the the `LOCUSINFO.EVENTS.SELF_OBSERVING` event? In such case the `isMoveTo` flag would remain `true` forever. Not sure if we need to handle such edge case..... we would need to add some timeout if we wanted to handle it...

Reply:
This event is emitted from inside locusInfo but I am doubtful if there will be a scenario when the event won't be emitted. (Unless locus sends selfObserving flag wrongly)

However, what would be the timeout we can set for this? I thought of a minute but what if that is too less of a time to timeout and addMedia isn't even called by that time

Reply:
I don't know how long the timeout should be, maybe Locus team has something to say about this. This is not blocking for this PR, but maybe try to find out if we need to handle this edge case

Reply:
Sure. I'll have a TODO to pursue this

Reply:
Added a TODO for this